slice/validate.go

```diff
@@ -35,5 +30,55 @@ func (s *Split) init() error {
 		return err
 	}
 
+	if err := s.validateFilters(); err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
Since there's no intrinsic additional error handling these lines can be replaced with:

```suggestion
	return s.validateFilters()
```